src/components/organisms/AssetActions/Compute/index.tsx

```diff
@@ -409,7 +411,17 @@ export default function Compute({
     } catch (error) {
       await checkPreviousOrders(selectedAlgorithmAsset)
       await checkPreviousOrders(ddo)
-      setError('Failed to start job!')
+
+      const { message, result } = ocean.assets.checkCredential(
+        selectedAlgorithmAsset,
+        CredentialType.address,
+        accountId
+      )
+
+      !result
```

Comment:
issue(blocking): same as above. If this should check for `false` avoid entering with `undefined` as value.